job_templates/ci_job.xml.em

```diff
@@ -168,6 +172,12 @@ fi
 @[  if os_name in ['linux', 'linux-aarch64'] and turtlebot_demo]@
 export CI_ARGS="$CI_ARGS --ros1-path /opt/ros/$CI_ROS1_DISTRO"
 @[  end if]@
+if [ "${CI_ENABLE_SANITIZER_TYPE}" != "none" ]; then
```

Comment:
Nit: I would use an empty string instead of none to avoid having issues if by any chance we end up having CI_ENABLE_SANITIZER_TYPE not being set.

Reply:
I was unable to identify a good way to translate jenkins build parameter value from 'none' to ''. Given that there is currently no way to set this parameter to '' through UI, let's continue with the above logic